adagrams/game.py

```diff
@@ -1,13 +1,80 @@
 from random import randint
-
+letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
 def draw_letters():
-    pass
+    drawed_letters=[]
+    all_letters=[]
+    letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
+    for letter, quantity in letters_pool.items():
+        for i in range(quantity):
```

Comment:
Since we don't use `i` in the looping variable, we should replace it with `_`. Read more [here](https://stackoverflow.com/a/5893186)

```suggestion
        for _ in range(quantity):
```

Reply:
I'd prefer lines 7-9 be in a descriptively named helper function so that `draw_letters` only has the responsibility of drawing letters (and doesn't also have to create all letters pool). 

What if you created a helper function like `create_letter_pool` and then invoke that function inside `draw letters`?

Then lines 7-9 could just become `all_letters = create_letter_pool()`